Composer/packages/server/src/controllers/asset.ts

```diff
@@ -48,8 +50,24 @@ export async function getProjTemplatesV2(req: any, res: any) {
       templates = templates.concat(await getNpmTemplates());
     }
 
+    let sortedTemplateList: BotTemplate[] = [];
+
+    // Sort incoming template array and reassign display name based on sort list
+    templateSortOrder.forEach((tempSortEntry, index) => {
+      const templateIndex = templates.findIndex((template) => {
+        return template.id === tempSortEntry.generatorName;
+      })
+      if (templateIndex != -1){
+      templates[templateIndex].name = tempSortEntry.displayName;
```

Comment:
Can this be read from the BotTemplate somehow? Could it be added to the package manifest?

Reply:
Are you suggesting having an index in botTemplate and the sort function just populates that index instead of actually sorting what is returned to the client? That would be possible, I initially implemented it that way but it required a bit of additional logic to pass that index order from the BotTemplate obj to the index prop in the detailsList where this is later populated. I figured sorting led to less front end changes.

Reply:
As for the package manifest. We could add sort data, down the line, to the meta data for each package that is provided from the npm registry. Not sure if that is the best approach, it does open up changing sorting outside of Composer release schedule but does add a dependency from Composers visual layer to some meta data in individual npm packages that are external to Composer

Reply:
The sort data is fine to live in Composer, what I'm suggesting is 
1. Move the `templateSortOrder` const and this sorting function into another module and add a unit test.
2. I was just asking if we can put the display name in the package manifest.

Reply:
Ah I see, yeah I will create a task for that. I have handled (1)